src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_load_store_emitters.cpp

```diff
@@ -65,18 +62,126 @@ void jit_load_emitter::emit_isa(const std::vector<size_t> &in_idxs, const std::v
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    XReg src = XReg(in_idxs[0]);
+    XReg prc = XReg(aux_gpr_idxs[0]);
+    TReg dst = TReg(out_idxs[0]);
+    DReg dst_d = DReg(out_idxs[0]);
+    HReg dst_h = HReg(out_idxs[0]);
+    SReg dst_s = SReg(out_idxs[0]);
+
+    switch (load_num_) {
+        case 0:
+            break;
+        case 1:
+            h->ldr(dst_h, ptr(src, byte_offset_));
+            break;
+        case 2:
+            h->ldr(dst_s, ptr(src, byte_offset_));
+            break;
+        case 3:
+            h->ldr(dst_s, ptr(src, byte_offset_));
+            h->add_imm(prc, src, byte_offset_ + 2 * sizeof(uint16_t), h->X_DEFAULT_ADDR);
+            h->ld1(dst.h[2], ptr(prc));
+            break;
+        case 4:
+            h->ldr(dst_d, ptr(src, byte_offset_));
+            break;
+        default:
+            OV_CPU_JIT_EMITTER_THROW("Unexpected number of elements to load.");
+    }
+}
+
+template <cpu_isa_t isa>
+void jit_load_emitter::load_byte(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    XReg src = XReg(in_idxs[0]);
+    XReg prc = XReg(aux_gpr_idxs[0]);
+    TReg dst = TReg(out_idxs[0]);
+    BReg dst_b = BReg(out_idxs[0]);
+    HReg dst_h = HReg(out_idxs[0]);
+    SReg dst_s = SReg(out_idxs[0]);
+
+    switch (load_num_) {
+        case 0:
+            break;
+        case 1:
+            h->ldr(dst_b, ptr(src, byte_offset_));
+            break;
+        case 2:
+            h->ldr(dst_h, ptr(src, byte_offset_));
+            break;
+        case 3:
+            h->ldr(dst_h, ptr(src, byte_offset_));
+            h->add_imm(prc, src, byte_offset_ + 2 * sizeof(int8_t), h->X_DEFAULT_ADDR);
+            h->ld1(dst.b[2], ptr(prc));
+            break;
+        case 4:
+            h->ldr(dst_s, ptr(src, byte_offset_));
+            break;
+        default:
+            OV_CPU_JIT_EMITTER_THROW("Unexpected number of elements to load.");
+    }
+}
+
+template <cpu_isa_t isa>
+void jit_load_emitter::emit_isa(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    bool is_supported_precision = one_of(src_prc_, ov::element::f32, ov::element::i32, ov::element::f16, ov::element::i8, ov::element::u8) &&
+                                  (src_prc_ == dst_prc_ || one_of(dst_prc_, ov::element::f32, ov::element::i32));
+    OV_CPU_JIT_EMITTER_ASSERT(is_supported_precision, "Unsupported precision pair.");
+    OV_CPU_JIT_EMITTER_ASSERT(load_num_ <= static_cast<int>((get_vec_length() / dst_prc_.size())),
+                              "Unexpected number of elements to load.");
+
+    switch (src_prc_) {
+        case ov::element::f32:
+        case ov::element::i32:
+            load_qbyte<isa>(in_idxs, src_prc_ == dst_prc_ ? out_idxs : aux_vec_idxs);
```

Comment:
Do we really need to load in `aux_vec_idxs` in this case?
Can `convert_emitter` handle tte same registers on input and output itself?

Reply:
Absolutely! Double checked, these conversion instructions support in place computation. Applied. Thanks Alexandra!

Reply:
Great! Thank you for the applying!

Reply:
@a-sidorova Hi Alexandra! My previous judgement might be wrong!

I reverted the commit "Remove unnecessary aux_vec_idxs(https://github.com/openvinotoolkit/openvino/pull/25815/commits/28af19b0481bae6f0af4829e48da52b9234e2176)", because it causes failures in test suite smoke_LSTMCellCommon. I checked some of these cases, they contains f16->f32 and f32->f16 conversions. Though such conversion cases are also covered by test suite smoke_Snippets_Convert, which successfully passed tests. I believe it is not safe to use inplace register for conversion instructions. Besides, the manual has not explicitly said these conversion instructions support inplace. https://developer.arm.com/documentation/ddi0602/2023-12/SIMD-FP-Instructions/FCVTL--FCVTL2--Floating-point-Convert-to-higher-precision-Long--vector--?lang=en. Please feel free to have further discussions.

Reply:
I really believe that it's possible because I found [implementations](https://github.com/ARM-software/ComputeLibrary/blob/f1929dc994d8e5afae5c77ca66446344119a8592/src/core/NEON/kernels/arm_gemm/indirect-interleaves/a64_interleave8_block1_fp16_fp32.hpp#L92) in ACL which use the same instructions (isa as well) and save the result to the same register. So probably there is another bug 🤔 
I'd suggest to come back to this question in bounds of the ticket 150430 at least

Reply:
Agree. Will come back to this question in bounds of the ticket 150430, as aux_vec_idxs is meant to be removed, when conversion emitter is separated from load/store emitters. Thanks Alexandra!